extensions/kusto/src/dashboard/bookExtensions.ts

```diff
@@ -0,0 +1,136 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I don't think this should be included here

Reply:
Removed